constellations/gemini-admin/src/semester/semesterEdit.js

```diff
@@ -147,16 +148,32 @@ export class SemesterEditPage extends React.Component {
                 {modalDiv}
                 <h2>{title}</h2>
 
-                <h4>Semester ID</h4>
-                <input
+                <InputText
+                    label="Semester ID"
+                    required={true}
                     value={this.state.inputSemesterId}
-                    onChange={(e) => this.handleChange(e, "inputSemesterId")}
+                    onChangeCallback={(e) =>
+                        this.handleChange(e, "inputSemesterId")
+                    }
+                    validators={[
+                        {
+                            validate: (text) => text != "",
+                            message: "You must input a semester ID",
+                        },
+                    ]}
                 />
 
-                <h4>Title</h4>
-                <input
+                <InputText
+                    label="Title"
+                    required={true}
```

Comment:
give description to user that the title should reflect the semesterId.
So if the semesterId is: 2020_fall, the title should be "Fall 2020"

Maybe it's worth auto-generating this for them?

Reply:
On second thought, let's do this. When a user enters the semesterId, let's immediately render what the semester Title could be. Since the semesterId is very predictable (year_season) all lowercase, We should be able to easily separate it into `Season year`

i.e.
2020_fall => Fall 2020
2021_winter => Winter 2021
2023_summer => Summer 2023